packages/client/src/entity-actions/entity-action-baked.ts

```diff
@@ -0,0 +1,85 @@
+import { EntityActionRosterEditEntry } from '../components/entity-action/entity-action-baked/entity-action-roster-edit-entry';
+import { EntityActionRosterAddEntry } from '../components/entity-action/entity-action-baked/entity-action-roster-add-entry';
+import { EntityActionRosterDownloadCsvTemplate } from '../components/entity-action/entity-action-baked/entity-action-roster-download-csv-template';
+import {
+  EntityActionColumnButtonBakedProps,
+  EntityActionColumnItem,
+  EntityActionTableButtonBakedProps,
+  EntityActionTableItem,
+} from './entity-action.types';
+import { EntityActionRosterExportCsv } from '../components/entity-action/entity-action-baked/entity-action-roster-export-csv';
+import { EntityActionRosterUploadCsv } from '../components/entity-action/entity-action-baked/entity-action-roster-upload-csv';
+
+export type EntityActionBakedTableItemIds =
+  | 'actionRosterAddEntry'
+  | 'actionRosterDownloadCsvTemplate'
+  | 'actionRosterExportCsv'
+  | 'actionRosterUploadCsv';
+
+export const entityActionBakedTableItems: {
+  [K in EntityActionBakedTableItemIds]: EntityActionTableItem;
+} = {
+  actionRosterAddEntry: {
+    type: 'table-button-item-baked',
+    id: 'actionRosterAddEntry',
+    entityType: 'roster',
+    displayName: 'Add Roster Entry',
+    operation: undefined,
+    requiredPermissions: ['canManageRoster'],
```

Comment:
Could these permissions be part of an enumeration somewhere to avoid hard-coded strings?